tests/test_result_to_dict.py

```diff
@@ -35,6 +35,13 @@ class Data(Base):
     data_enum = Column(Enum(MyEnum))
 
 
+def convert_str_to_dict(result):
+    obj = result.get('object')
+    if obj and isinstance(obj, str):

```

Comment:
The following will require to write a new test: 

`result_to_dict` need to return a value in order for it to work. 
if you look at the file `result_to_dict.py` line `111`
```
    if callback:
        results = callback(results)
```

when the call back return None it will override the entire result 
what do you think about this 

```python
    if callback:
        results = callback(results) or results
```